<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemInfo</class>
 <widget class="QWidget" name="SystemInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(115, 210, 22);
font-size: 24px;</string>
  </property>
  <widget class="QLabel" name="lbl_overview">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Overview</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_network">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Network</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_wifi">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>WiFi</string>
   </property>
  </widget>
  <widget class="QLabel" name="wifi_mac">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>66:66:66:66:66:66</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_eth">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>100</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Eth0</string>
   </property>
  </widget>
  <widget class="QLabel" name="eth_mac">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>250</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Placeholder</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gps">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>GPS</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_latitude">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="latitude">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>48.137154</string>
   </property>
  </widget>
  <widget class="QLabel" name="longitude">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>11.576124</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_longitude">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="altitude">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Placeholder</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_altitude">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_system">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>270</y>
     <width>120</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>System</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_cpu_temp">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CPU Temp</string>
   </property>
  </widget>
  <widget class="QLabel" name="cpu_temp">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>42Â°C</string>
   </property>
  </widget>
  <widget class="QLabel" name="eth_ip">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Placeholder</string>
   </property>
  </widget>
  <widget class="QLabel" name="wifi_ip">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>200</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>255.255.255.255</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>360</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_boot">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Boot</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_run">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time_current">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_run">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>420</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_boot">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="time_current">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>150</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>31.12.2020</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_date">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>130</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLabel" name="gps_valid">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>current</string>
   </property>
  </widget>
  <widget class="QLabel" name="gps_timestamp">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>current</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
